packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -26,6 +27,26 @@ class ConfigCommand extends FlutterCommand {
       negatable: false,
       hide: !verboseHelp,
       help: 'Print config values as json.');
+    argParser.addFlag(
+      flutterWebFeature.setting.configSetting,
+      help: 'Enable or disable flutter web on master and dev channels.',
```

Comment:
Drive-by: these help strings seem very likely to get out of sync with the actual channel settings over time. I'd suggest either gating the `addFlag` behind a check that's based on querying availability from the `*Feature` object, or (if you want it available on all channels as in this version) generating an available-channel string programatically from the `Feature`.

Reply:
Good point, I've added some logic to generate the help messages from the features themselves. I've also intentionally kept the flags present in all channels to avoid issues with tooling/scripts/users needing to keep track of which flags are defined where